core/package.json

```diff
@@ -64,6 +64,7 @@
     "jest": "^29.7.0",
     "jest-cli": "^29.7.0",
     "prettier": "^2.6.1",
+    "puppeteer": "21.1.1",
```

Comment:
Why do we need puppeteer as a dev dependency?

Reply:
Stencil regressed this behavior in v3.3.0: https://github.com/ionic-team/stencil/issues/4526.

At this point it is required or else you get a build error when `tsc` runs.

Reply:
Does that mean this can be removed once the issue is fixed on Stencil's end?

Reply:
Correct, created FW-5617 to track.